### ISSUE TITLE
about train/val/test dataset 

### ISSUE THREAD

**Issue by love112358** [eezkni/UEGAN#8]
I downloaded the dataset from https://drive.google.com/drive/folders/1Jv0_9CnYxh_2ReFaVrwG19O3F7xBtdZT, which only contains expertC and original. expertc stands for label, original stands for raw, right? But there is no verification set and test set in this website. We need to deal with the verification set and test set , right

**Comment by eezkni** [comment]
Yes, expertC stands for label, and original stands for raw. 
In order to generate unpaired training data, the subset is randomly divided into three partitions: 1) the first partition has 2,250 raw photos as low-quality input; 2) the second partition consists of retouched version of another 2,250 raw photos and served as the desired high-quality photos; 3) the last partition is the remaining 500 raw photos used for validation (100 images) and testing (400 images). These three parts have no overlaps with each other.

**Comment by kandulapraveen** [comment]
Thanks for the information. Can you please release the names of the images which are low-quality input, high-quality photos, validation and test images? This will make it easy for a faithful comparison infuture.
